### DISCUSSION TITLE
Changing the order of rows in smartscreen

### DISCUSSION THREAD

**Discussion by levnet1** [jellyfin/jellyfin-androidtv#2813] [General]
Hey is there an easy way to make the "views" row appear first when going into movies or tv library? 
To clarify i mean in android studio (im new to this and have done minor things like change logo and colors)

**Comment by nielsvanvelzen** [comment]
Right now you can only change the order of the rows on the home screen. The smart screens cannot be changed.

**Reply by levnet1** [reply]
Do you mean they cannot be changed while using the app? What im asking is which file I can edit in android studio to change the order of the rows in smart screen. That must be possible right?

**Comment by levnet1** [comment]
i figured it out, i had to edit the enhancedbrowserfragment
i just moved the add aditional rows line higher than the other rows and it worked.

**Reply by nielsvanvelzen** [reply]
Sorry I thought you wanted to know if there was a preference for it. As you said it's an easy change to just switch the order of row creation.

I'd like to remind you of our [license](https://github.com/jellyfin/jellyfin-androidtv/blob/master/LICENSE) and kindly ask you to make the source of your changes public. Thank you.
